test/make-auto.d/unpack.bats.in

```diff
@@ -0,0 +1,11 @@
+@test 'unpack %(tag)s' {
+    scope %(scope)s
+    %(arch_exclude)s
+    prerequisites_ok %(tag)s
+    if [[ -e "${ch_tardir}/%(tag)s".sqfs ]]; then
+        unsquashfs -d "${ch_imgdir}/%(tag)s" "${ch_tardir}/%(tag)s".sqfs
```

Comment:
Tests outside of the 'auto' realm sometimes require the image to be unpacked. I spoke of this during last weeks meeting I think. I proposed having someone run down all the tests that currently require a writable image unpacked, along with other images that rely on these unpacked directories. For former would still need to unpack. The latter could possibly be changed to simply expect a squashfs instead then mount, run, and unmount. Should I open a separate issue regarding this?

Reply:
Yeah, why don't we leave that for another issue. I wonder if the tests that need an unpacked (writeable) image could do the unpack themselves in the squashfs case.

Reply:
That's what I was thinking. This way if that test has a specific dependency it has to satisfy that itself, instead of relying on the build phase unpacking everything.